.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,105 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'master'
+
+jobs:
+    test:
+        name: 'PHP ${{ matrix.php-version }} ${{ matrix.dependencies }} ${{ matrix.symfony-require }}'
+
+        runs-on: ubuntu-20.04
+
+        services:
+            mysql:
+                image: mysql:5.7
+                env:
+                    MYSQL_DATABASE: phpcr_odm_tests
+                    MYSQL_USER: user
+                    MYSQL_PASSWORD: password
+                    MYSQL_ROOT_PASSWORD: rootpassword
+                ports:
+                    - 33306:3306
+                options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+        env:
+            SYMFONY_REQUIRE: ${{ matrix.symfony-require }}
+
+        strategy:
+            matrix:
+                php-version:
+                    - '7.1'
+                    - '7.2'
+                    - '7.3'
+                    - '7.4'
+                    - '8.0'
+                dependencies: [highest]
+                symfony-require: ['']
+                transport: [doctrine_dbal, jackrabbit]
```

Comment:
looking at the build it looks like something is not working with the `transport`. can you accept the suggestion to put the transport into the name? maybe that shows us what the problem is.

Reply:
oh, interesting. i was able to accept my own suggestion into your PR :open_mouth:

Reply:
looking at the build in your repo, it seems we get a 3rd type where transport is not set: https://github.com/jordisala1991/phpcr-odm/runs/4262215760?check_suite_focus=true

i think this is because of the `include` section. i think you need to add a transport in there as well - in the travis config we duplicated each entry to have one with each transport.

Reply:
Yep, I coudn't finish it earlier, I will try to find time this weekend.